test/alternator/test_manual_requests.py

```diff
@@ -174,6 +174,18 @@ def test_tags_return_empty_body(dynamodb, test_table):
     response = requests.post(req.url, headers=req.headers, data=req.body, verify=False)
     assert not response.text
 
+def test_create_table_without_keyschema(dynamodb, test_table_s):
```

Comment:
Please add a comment above the test including at least (and I'm ok with including only) a reference to the issue this test reproduces. For example:
```
# Reproduces issue #16233
```
These references are useful if a test starts to fail and we want to know which bug returned - or which PR we forgot to backport.